src/core/ext/filters/client_channel/resolver/polling_resolver.h

```diff
@@ -116,6 +110,12 @@ class PollingResolver : public Resolver {
     kReresolutionRequestedWhileCallbackWasPending,
   };
   ResultStatusState result_status_state_ = ResultStatusState::kNone;
+  /// next resolution timer
+  absl::optional<grpc_event_engine::experimental::EventEngine::TaskHandle>
+      next_resolution_timer_handle_{absl::nullopt};
```

Comment:
No need to explicitly initialize to `absl::nullopt`; that's the default for `absl::optional<>`.